app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
```

Comment:
`skills` variable should be an attribute of the instance not class.

Reply:
don't forgot about [checklist.md](https://github.com/mate-academy/py-software-engineers/blob/master/checklist.md)